packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -0,0 +1,281 @@
+
+> NOTE: NEEDS DEV REVIEW & VISUAL DESIGN?
+
+`UP ARROW` on the focused component opens the menu and focuses (but does not select) the first or last menu item.
+
+![ ](spec-images/select-29.png)
+
+#### Menu
+
+> QUESTION: WHAT KEYBOARD SHORTCUT SHOULD BE USED TO FOCUS THE FIRST MENU ITEM AFTER THE FILTER BAR?
+
+| Key                        | Description                                                                                                                                                               |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ENTER` or `SPACE`         | Sets the value of the field button to the focused menu item, closes the menu, and moves focus to the select                                                               |
+| `DOWN ARROW` or `UP ARROW` | Moves visual focus to the next or previous menu item                                                                                                                      |
+| `ESC`                      | Closes the menu and moves visual focus to the select                                                                                                                      |
+| `HOME` or `END`            | Moves visual focus to the first or last menu item                                                                                                                         |
+| `PAGEUP` or `PAGEDOWN`     | Jumps visual focus up or down ten menu items (or to first or last menu item)                                                                                              |
+| _Printable characters_     | Moves visual focus to the filter input (if filterable) with the characters typed or to the first menu item (if not filterable) that matches the typed character or string |
+
+> NOTE: NEED VISUAL DESIGN TO DETERMINE WHETHER BOTH FOCUS AND SELECTION ARE VISUALLY INDICATED ON MENU ITEMS
+
+From a focused menu item, `DOWN ARROW` moves focus and selects the next menu item. If the focused/selected menu item is the last in the menu, `DOWN ARROW` does nothing (not pictured).
+
+![ ](spec-images/select-30.png)
+
+From a focused menu item, `UP ARROW` moves focus and selects the previous menu item. If the focused/selected menu item is the first in the menu, `UP ARROW` does nothing (not pictured).
+
+![ ](spec-images/select-31.png)
+
+`ENTER` on a focused/selected menu item commits the value, updates the field button, and closes the menu.
+
+![ ](spec-images/select-32.png)
+
+`ESC` on a focused/selected menu item cancels the value and closes the menu.
+
+![ ](spec-images/select-33.png)
+
+## Future Considerations
+
+-   Should the select component allow multi-selection? Users on touch-screen devices may see a truncated list of comma-separated selections in the field button, without the ability to hover for a tooltip.
```

Comment:
> Should the select component allow multi-selection?

Can we remove this as a Future Consideration since we recommend above that clients wanting multiselect should use the tag picker instead?